internal/models/questions/questions.go

```diff
@@ -133,13 +147,57 @@ func GetQuestionsByQuizID(db *sql.DB, id *uuid.UUID) (*[]Question, error) {
 	return scanQuestionsFromFullRows(db, rows)
 }
 
+// Gets nth question in a givnen quiz, indexing from 1.
+func GetNthQuestionByQuizId(db *sql.DB, quizId uuid.UUID, questionNumber uint) (*Question, error) {
+	row := db.QueryRow(
+		`SELECT
+				q.id, q.question, q.image_url, q.arrangement, q.article_id, q.quiz_id, q.time_limit_seconds, q.points
+			FROM
+				questions q
+			WHERE
+				quiz_id = $1
+			GROUP BY
+				q.id
+			ORDER BY
+				q.arrangement ASC
+			LIMIT 1
+			OFFSET $2;`, quizId, questionNumber-1)
+
+	return scanQuestionFromFullRow(db, row)
+}
+
+// Gets the next question in a quiz passed questionId belongs to. If no more questions, sql.ErrNoRows  will be returned
```

Comment:
`Gets the next question in a quiz passed questionId belongs to` this text is a bit confusing. Maybe `Gets the next question in a quiz after the current question (question with the given questionId)` is a bit clearer?